lib/vmdb/loggers.rb

```diff
@@ -76,6 +80,37 @@ def self.create_logger(log_file_name, logger_class = VMDBLogger)
       nil
```

Comment:
huh. I was not expecting a `nil` here but something more like:

```suggestion
      ManageIQ::Loggers::Journald.new
```

Reply:
I don't follow - The reason it's a nil here is specifically because we _can't_ create a Journald logger

Reply:
hmm. not sure if I follow either